lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -91,7 +105,8 @@ struct ImportTracker {
   FailureOr<llvm::SmallSet<StringRef, 4>> getLibraryNames(ModuleOp module) {
     auto walkRes = module.walk([&](ComponentOp component) {
       for (auto &op : *component.getBody()) {
-        if (!isa<CellInterface>(op) || isa<InstanceOp>(op))
+        if (!isa<CellInterface>(op) || isa<InstanceOp>(op) ||
```

Comment:
This is the way it was before, I just added the PrimitiveOp check. I assume this is because all primitives have a cell interface, so if there is no CellInterface we don't need to worry about imports. This is a little confusing because the PrimitiveOp is also not a core calyx primitive so it does not need to be considered for Calyx imports either.